packages/ERTP/test/unitTests/mathHelpers/test-natMathHelpers.js

```diff
@@ -1,4 +1,4 @@
-// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
I think we're being inconsistent about whether we are keeping this line or not. We kept it for test-splitPayments.js

Reply:
True. eslint ddn't complain so I didn't notice. I'll delete it from the others.

Reply:
oh, in fact this test doesn't need to be run under SES anyways, it's only the ones that use swingset, spawner, or zoe's contract-evaluation functions which need `install-ses`. I'll restore it back to the original state.